package.json

```diff
@@ -52,15 +51,15 @@
   },
   "devDependencies": {
     "code": "^1.4.1",
-    "eslint": "^2.10.2",
-    "eslint-config-semistandard": "^5.0.0",
-    "eslint-config-standard": "^4.4.0",
-    "eslint-plugin-standard": "^1.3.1",
-    "lab": "^5.11.1",
+    "eslint-config-semistandard": "6.0.2",
+    "eslint-config-standard": "5.3.5",
+    "eslint-plugin-promise": "1.3.2",
+    "eslint-plugin-standard": "1.3.3",
+    "lab": "10.9.0",
     "nodemon": "^1.3.7",
-    "npm-shrinkwrap": "200.5.1",
     "prompt": "^0.2.14",
     "proxyquire": "^1.4.0",
+    "selenium-webdriver": "2.47.0",
```

Comment:
chose this version based on [Sauce Labs docs](https://wiki.saucelabs.com/display/DOCS/Instant+Selenium+Node.js+Tests)